v2/internal/testcommon/kube_test_context_envtest.go

```diff
@@ -37,7 +37,7 @@ func createSharedEnvTest(cfg testConfig, namespaceResources *namespaceResources)
 	environment := envtest.Environment{
 		ErrorIfCRDPathMissing: true,
 		CRDDirectoryPaths: []string{
-			"../../config/crd/bases",
+			"../../config/crd/packed",
```

Comment:
Oh, I guess the benefit of putting the patched CRD output somewhere under `config/crd` is that we can hardcode it here rather than trying to feed it in as a parameter somehow. That makes sense.